browser/src/core/Socket.js

```diff
@@ -1526,7 +1526,7 @@ app.definitions.Socket = L.Class.extend({
 
 	_onSocketClose: function () {
 		window.app.console.debug('_onSocketClose:');
-		if (!this._map._docLoadedOnce) {
+		if (!this._map._docLoadedOnce && this.ReconnectCount === 0) {
```

Comment:
I noticed that `ReconnectCount` is only increased in our code.
What about the case when we successfully reconnected to the server? shouldn't it be reset to 0 ?

Reply:
Yes it resets to zero here
https://github.com/CollaboraOnline/online/blob/3f46c1db445b9840318af0a4994f0c3ff1012285/browser/src/core/Socket.js#L1090

Reply:
ah I missed that one